guide/getting-started/hardware.md

```diff
@@ -0,0 +1,96 @@
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/hardware/hardware-seed-storage.jpg"
+   retina = "/assets/images/guide/getting-started/hardware/hardware-seed-storage@2x.jpg"
+   alt-text = "Examples of seed storage options"
+   width = 800
+   height = 400
+%}
+
+## Hardware wallets
+
+You may already be familar with key fobs from your bank or work. They are small devices with specific built-in authentication that is used to control and secure access to a computer system. Key fobs are not connected to the internet or any other network, so they cannot easily be hacked without physical access.
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/hardware/hardware-wallets.jpg"
+   retina = "/assets/images/guide/getting-started/hardware/hardware-wallets@2x.jpg"
+   alt-text = "Examples of hardware wallets"
+   caption = '<a href="https://coldcardwallet.com" target="_blank">Coldcard</a>, <a href="https://www.ledger.com/" target="_blank">Ledger</a>, <a href="https://trezor.io" target="_blank">Trezor</a>, and <a href="https://foundationdevices.com" target="_blank">Foundation</a>'
+   width = 800
+   height = 400
+%}
+
+Hardware wallets for bitcoin play a similar role but with several differences. Similar to key fobs, they securely store an authentication mechanism and isolate it from other devices and networks. In the case of bitcoin, these are the seed, associated keys and all functions that require direct interaction with them.
+
+Because hardware wallets are so optimized for this specific uses case, they have very small screens and limited input options. Most interactions with them happen via desktop software that is also provided by the manufacturer. So users perform most wallet interactions on their desktops, and the application sends only specific requests to the hardware wallet to verify or sign data. This provides a great balance of security and convenience, and keeps hardware wallets generally affordable for most use cases.
+
+## Nodes
+
+Thousands of computers participate in the Bitcoin network. They are called nodes and continuously exchange, verify and compare transaction data. Running your own node, instead of relying on a third-party node, has the benefit that transactions are getting validated by your own personal device. Nodes also improve privacy and help secure the network. Several DIY and consumer hardware products have been created to make it easy to run your own node on a dedicated device.
```

Comment:
Thousands of computers participate in the Bitcoin network. They are called nodes and continuously exchange, verify and compare transaction and block data. Running your own node, instead of relying on a third-party node, has the benefit that transactions are getting validated by your own personal device. Nodes also improve privacy and help secure the network. Several DIY and consumer hardware products have been created to make it easy to run your own node on a dedicated device.

Reply:
@ConorOkus yep, still didn't do it right, sorry 😳

Reply:
There is no right way to do this so no worries, was just a suggestion 👍

Reply:
Added the changes. Thank you.